common/src/utils/local_file_path.rs

```diff
@@ -0,0 +1,18 @@
+use std::path::PathBuf;
+
+/// It will work to load local files in img or video tags, but will ignore drive
+const PREFIX_TO_WORK_ON_WINDOWS_OS: &str = "http://dioxus.";
```

Comment:
Does this only work on windows or does it work across all platforms? If so, maybe we can add the prefix to the path for loading the local file in the webview (though i believe you said this was webview specific but dont recall if it was ever specific to a single platform)